ansible/files/bootstrap/telemetry.sh

```diff
@@ -62,6 +26,14 @@ while [ $# -gt 0 ]; do
             shift
             TIMEOUT=$1
             ;;
+        --email)
+            shift
+            EMAIL=$1
+            ;;
+        --efs-dns)
+            shift
+            EFS_DNS=$1
```

Comment:
@fbertsch Can you clarify what "dev" means in this instance? ATMO has stage and prod environments, no official dev environment (yet).

Since the most pressing issues with ATMO (and Airflow etc) are blocked by having a [common set of EMR scripts](https://github.com/mozilla/telemetry-analysis-service/issues/203) I'm going to be moving the content of this repo into the one of ATMO. That means I'll be able to hardcode the EFS URIs into ATMO but keep them configurable. I just need to know how the two values you mention map to the dev/stage/prod environments of ATMO.